frontend/model/state.js

```diff
@@ -157,6 +157,28 @@ sbp('sbp/selectors/register', {
           upgradeAction(contractID)
         })
       }
+    })();
+    (() => {
+      // Update expired invites
+      // If fewer than 150 'anyone can join' have been used, create a new
+      // 'anyone can join' link up to 150 invites
+      const ourIdentityContractId = state.loggedIn?.identityContractID
+      if (!ourIdentityContractId || !state[ourIdentityContractId]?.groups) return
+      Object.entries(state[ourIdentityContractId].groups).map(([groupID, { hasLeft }]: [string, Object]) => {
+        const groupState = state[groupID]
+        if (hasLeft || !groupState?.invites) return undefined
+        const now = Date.now()
+        const usedInvites = Object.keys(groupState.invites)
+          .filter(invite => groupState.invites[invite].creatorID === INVITE_INITIAL_CREATOR)
+          .reduce((acc, cv) => acc +
+        (!(groupState._vm.invites[cv].expires >= now)
+          ? groupState._vm.invites[cv].initialQuantity
+          : ((groupState._vm.invites[cv].initialQuantity - groupState._vm.invites[cv].quantity)
+            ) || 0), 0)
+        return (usedInvites < 150) ? groupID : undefined
```

Comment:
Everywhere we use this value `150`, could you replace it with `MAX_GROUP_MEMBER_COUNT`?

Reply:
Sure